test/runtests.jl

```diff
@@ -111,12 +114,12 @@ end
     @test_throws ArgumentError ZlibDeflateOutputStream(UInt8[], level=10)
     @test_throws ArgumentError ZlibInflateOutputStream(UInt8[], bufsize=0)
 
-    deflated = read(ZlibDeflateInputStream("foo".data))
+    deflated = read(ZlibDeflateInputStream(Vector{UInt8}("foo")))
     buf = IOBuffer()
     out = ZlibInflateOutputStream(buf)
     BufferedStreams.writebytes(out, deflated, length(deflated), true)
     flush(out)
-    @test takebuf_string(buf) == "foo"
+    @test String(take!((buf))) == "foo"
```

Comment:
Even @ViralBShaw has mentioned how big a pain it is to try to determine the minimal compat version......